package.json

```diff
@@ -309,14 +339,14 @@
         "stylelint-config-standard": "^19.0.0",
         "stylelint-order": "^3.1.1",
         "tabbable": "^1.1.3",
-        "ts-loader": "^6.2.1",
+        "ts-loader": "^9.5.1",
         "typescript": "^5.2.2",
         "uniqid": "^5.2.0",
         "uuid": "^8.3.2",
         "wait-on": "^3.3.0",
-        "webpack": "^4.41.5",
+        "webpack": "^5.89.0",
         "webpack-bundle-analyzer": "^3.6.0",
-        "webpack-cli": "^3.3.10",
+        "webpack-cli": "^4.9.1",
```

Comment:
This upgrade is required due to receiving the error in https://github.com/webpack/webpack-cli/issues/3005, after upgrading webpack and babel-loader

There are higher versions available, but I decided to use the lowest version which includes the fix to avoid any issues. https://github.com/webpack/webpack-cli/releases/tag/webpack-cli%404.9.1

If we want to upgrade more, I think we should do it in a separate PR.